xchange-bx/src/main/java/org/knowm/xchange/bx/service/BxDigest.java

```diff
@@ -0,0 +1,50 @@
+package org.knowm.xchange.bx.service;
+
+import org.knowm.xchange.service.BaseParamsDigest;
+import si.mazi.rescu.RestInvocation;
+
+import javax.ws.rs.FormParam;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+class BxDigest extends BaseParamsDigest {
```

Comment:
You can avoid manual hex conversion. Try to look into `org.knowm.xchange.liqui.service.LiquiDigest` class. There is used `javax.xml.bind.DatatypeConverter` with string and hex conversions.